src/utils/permissions-utils.js

```diff
@@ -80,7 +80,7 @@ function authorizeEntityManipulation(entity, sender, senderPermissions) {
   } else if (template.endsWith("-camera")) {
     return isCreator || senderPermissions.spawn_camera;
   } else if (template.endsWith("-pen")) {
-    return isCreator || senderPermissions.spawn_pen;
+    return isCreator || senderPermissions.spawn_drawing;
```

Comment:
Did the name of this change elsewhere?

Reply:
no, this was incorrectly named in the first place, and was causing ownership transfer to get busted with pens.